UnityProject/Assets/Scripts/Player/PlayerSync.cs

```diff
@@ -443,12 +449,19 @@ public void ServerTryEscapeContainer()
 
 	private void UpdateMe()
 	{
+		inputMovementDetected = false;
 		if (isLocalPlayer && playerMove != null)
 		{
 			if (PlayerManager.MovementControllable == this as IPlayerControllable)
 			{
 				didWiggle = false;
-				if (KeyboardInputManager.IsMovementPressed() && Validations.CanInteract(playerScript,
+				bool inputDetected = KeyboardInputManager.IsMovementPressed();
+				if (inputDetected == true && inputMovementDetected != true)
+				{
+					inputMovementDetected = inputDetected;
+					CmdSetMovementInputState(inputDetected);
+				}
```

Comment:
```suggestion
				bool inputDetected = KeyboardInputManager.IsMovementPressed();
				if (inputDetected != lastInputState)
				{
					lastInputState = inputDetected;
					CmdSetMovementInputState(inputDetected);
				}
```

Using another bool explains what i was trying to do